src/search/landmarks/landmark_heuristic.h

```diff
@@ -31,7 +31,7 @@ class LandmarkHeuristic : public Heuristic {
     std::unique_ptr<LandmarkStatusManager> lm_status_manager;
     std::unique_ptr<successor_generator::SuccessorGenerator> successor_generator;
 
-    void initialize(const plugins::Options &opts);
+    void initialize(const plugins::Options &initialize_options);
     void compute_landmark_graph(const plugins::Options &opts);
```

Comment:
special cases that still use `Options` inside the class. Would be good to mark this with a comment.